### ISSUE TITLE
Add fields for control property

### ISSUE THREAD

**Issue by rappen** [Power-Maverick/PCF-CustomControlBuilder#31]
It would be nice to be able to specify properties for this line from the manifest within the tool:
```
<property name="sampleProperty" display-name-key="Property_Display_Key" description-key="Property_Desc_Key" of-type="SingleLine.Text" usage="bound" required="true" />
```

---
- Deployment: Online
- DB Version: 9.1.0.25551
- Connection Controls Version: 1.2020.10.35
- XrmToolBox Version: 1.2020.11.42
- Tool Version: 2.2020.10.212

**Comment by rappen** [comment]
Shouldn't Groot have a capital G? It's his name, after all...

**Comment by Power-Maverick** [comment]
Fixed the name on Groot. Thanks @rappen I like the idea

**Comment by rappen** [comment]
I liked Groot. 
Now he's just being "see me see me see me!!" which is kinda annoying.... 😉

**Comment by Power-Maverick** [comment]
Resolved in version: 2.2021.05.241
